app/javascript/actions/initialState.js

```diff
@@ -99,8 +99,21 @@ export function loadInitialState(initialState) {
     result[types[index]] = shaped;
     return result;
   }, {});
+  // Look for questions inside building_types
```

Comment:
Discussed with @ethanlee16 that this handles the case for when questions are created that aren't yet associated with a building. On initial load it was looking for questions inside buildings, so that's why it seemed like the questions weren't in the store when reloading the page.